drizzle/0000_cynical_the_hood.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS "advocates" (
+	"id" serial PRIMARY KEY NOT NULL,
+	"first_name" text NOT NULL,
+	"last_name" text NOT NULL,
+	"city" text NOT NULL,
+	"degree" text NOT NULL,
+	"payload" jsonb DEFAULT '[]'::jsonb NOT NULL,
+	"years_of_experience" integer NOT NULL,
+	"phone_number" bigint NOT NULL,
+	"created_at" timestamp DEFAULT CURRENT_TIMESTAMP
+);
+--> statement-breakpoint
+CREATE INDEX IF NOT EXISTS "advocate_city_idx" ON "advocates" USING btree ("city");--> statement-breakpoint
```

Comment:
Added these indexes as they seem like good low-hanging fruit. I don't want to over-engineer this too much upfront, so I would monitor performance of the queries on this table and adjust/add indexes if needed.